### PR TITLE
OPS: Enhance Catalyst build process

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/hp6gqvwhn5dqwx3mzgoogfe4ja?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11219)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/egpasmttjnw74csqdb4gdogzwq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11220)

File: scripts/maccatalystpatches/lightning-ldk-wallet.ts [hunk 1]
```diff
@@ -0,0 +1,469 @@
+import { BitcoinUnit, Chain } from '../../models/bitcoinUnits';
```
Comment (Overtorment) [inline scripts/maccatalystpatches/lightning-ldk-wallet.ts:1]: what exactly you had to fix in this class?

Comment (Overtorment) [reply scripts/maccatalystpatches/lightning-ldk-wallet.ts:1]: couldnt make a patch? 
next time i work on this class maccatalyst  builds will be fucked?

File: scripts/maccatalystpatches/torrific.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/**
```
Comment (Overtorment) [inline scripts/maccatalystpatches/torrific.js:1]: same question

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: scripts/maccatalystpatches/lightning-ldk-wallet.ts (hunk 1 above)
Comment (marcosrdz) [reply scripts/maccatalystpatches/lightning-ldk-wallet.ts:1]: I want to keep the rest of the codebase untouched but these files modified. Why? Rnldk and rn tor are not catalysts compatible thus importing crashes the app. By deleting the import and calls for these imports im able to keep the rest of the app as-is

Comment (Overtorment) [reply scripts/maccatalystpatches/lightning-ldk-wallet.ts:1]: maybe its better to do `cp` and then `patch` ? so when files change developer wont have to remember to replace  and edit them in `maccatalyst` folder?

Comment (marcosrdz) [reply scripts/maccatalystpatches/lightning-ldk-wallet.ts:1]: Doing a patch would require me to be in sync with the changes made on those files (line numbers would matter). Compared to simply replacing them with empty functions.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/fnx2ysx4vaejypgjqx3mwudna4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11222)

File: scripts/maccatalystpatches/lightning-ldk-wallet.ts (hunk 1 above)
Comment (Overtorment) [reply scripts/maccatalystpatches/lightning-ldk-wallet.ts:1]: ok yolo. lets merge.
and future pain in the ass it produces will be on you.

**Pull request merged**